### PR TITLE
Android Onboarding

### LINKED ISSUE #220: Android Onboarding
- [x] Sample application link `sdk-examples` repo
- [x] How to download okapi native binaries
- [x] Walkthrough of server and okapi

### PR DESCRIPTION
**Pull request by lucasamonrc**

### CODE REVIEW

File: docs/android/drivers-license-android.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+val accountService = AccountService(null, config)
+
+val walletService = WalletService(null, config)
+val credentialsService = CredentialsService(null, config)
+```
+
+## Setup wallet profiles
+
+Let's create three different profiles, each pointing to a separate wallet. Since we are using a single console app for this demo, we will simply set the active profile before each interaction to designate which actor is currently taking action.
+To create a new wallet profile, we use the [Create Wallet](../reference/services/wallet-service/#create-wallet) feature.
```
Comment (michaeldboyd) [inline docs/android/drivers-license-android.md:34]: This should use the account signin feature now

File: docs/android/drivers-license-android.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    val itemId = walletService.insertItem(credential)
+    println("item id = $itemId")
+
+    return itemId
+}
+```
+
+## Proof of Driver's License
+
+If Allison is ever stoped by a police officer, she must show her driver's license. The request for this proof also comes in a form of JSON frame.
```
Comment (michaeldboyd) [inline docs/android/drivers-license-android.md]: `stopped` - typo

**Review by fundthmcalculus — APPROVED** [review]
looks good! Eventually, we need to do code injection, but that's harder since it's a cross-repo injection.

File: mkdocs.yml [hunk 3]
```diff
@@ -24,7 +24,7 @@ nav:
     - Java: java/index.md
     - Web: web/index.md
     # - iOS: ios/index.md
-    # - Android: android/index.mdw
+    - Android: android/index.mdw
```
Comment (michaeldboyd) [inline mkdocs.yml]: what is .mdw?

File: docs/android/drivers-license-android.md (hunk 1 above)
Comment (michaeldboyd) [reply docs/android/drivers-license-android.md:34]: Doesn't need to be done in this PR, but we need to add a reference section for the account service feature

File: docs/android/drivers-license-android.md (hunk 2 above)
Comment (fundthmcalculus) [inline docs/android/drivers-license-android.md]:
```suggestion
If Allison is ever stopped by a police officer, she must show her driver's license. The request for this proof also comes in a form of JSON frame.
```

File: mkdocs.yml (hunk 3 above)
Comment (fundthmcalculus) [inline mkdocs.yml]:
```suggestion
    - Android: android/index.md
```

File: docs/android/drivers-license-android.md (hunk 1 above)
Comment (fundthmcalculus) [reply docs/android/drivers-license-android.md:34]: tracked here: https://github.com/trinsic-id/sdk/issues/370

**Pull request merged**